lib/ui/compositing/scene.cc

```diff
@@ -23,6 +23,8 @@
 
 namespace flutter {
 
+static constexpr float kFallbackPixelRatio = 2.0f;
```

Comment:
This should probably have a comment answering questions like: Why is this 2.0? When do we fallback to this? Is this just meant to support legacy behavior and new logic should not use this constant? Or when should new logic use this constant?

Reply:
(I see that this is kinda explained on defaultViewPixelRatio, might be worthwhile to link to that here)